.cirrus.yml

```diff
@@ -15,6 +15,7 @@ task:
     PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1 # disable auto-browser download
     KEYRING: /usr/share/keyrings/nodesource.gpg
     NODE_VERSION: node_14.x # set node version e.g: "node_16.x" for Node 16 LTS
+    SUDO_NON_ROOT_CMD: sudo --login --user=ranchertest
```

Comment:
Converted the sudo -iu to a env var, as we have many instances of the same command in the file.